keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperPlugin.kt

```diff
@@ -166,20 +167,21 @@ public class KeeperPlugin : Plugin<Project> {
       isCanBeConsumed = false
       isCanBeResolved = true
       defaultDependencies {
-        add(project.dependencies.create("com.android.tools:r8:$DEFAULT_R8_VERSION"))
-      }
-    }
-
-    val androidJarFileProvider = provider {
-      val compileSdkVersion = appExtension.compileSdkVersion
-          ?: error("No compileSdkVersion found")
-      File("${appExtension.sdkDirectory}/platforms/${compileSdkVersion}/android.jar").also {
-        check(it.exists()) {
-          "No android.jar found! Expected to find it at: $it"
+        val version = when (extension.enableTraceReferences.get()) {
+          false -> PRINTUSES_DEFAULT_VERSION
+          true -> TRACE_REFERENCES_DEFAULT_VERSION
         }
+        logger.debug("keeper r8 default version: $version")
+        add(project.dependencies.create("com.android.tools:r8:$version"))
```

Comment:
It's working fine. Just for reference, it's resolved just after the configuration phase.